sys/kern/build.rs

```diff
@@ -45,3 +54,110 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     }
     Ok(())
 }
+
+fn generate_statics() -> Result<(), Box<dyn std::error::Error>> {
+    let kconfig: KernelConfig =
+        ron::de::from_str(&env::var("HUBRIS_KCONFIG")?)?;
+    println!("cargo:rerun-if-env-changed=HUBRIS_KCONFIG");
+
+    let out = &PathBuf::from(env::var_os("OUT_DIR").unwrap());
+    let mut file = File::create(out.join("kconfig.rs")).unwrap();
+
+    writeln!(file, "// See build.rs for details")?;
+
+    writeln!(
+        file,
+        "const HUBRIS_FAULT_NOTIFICATION: u32 = {};",
+        kconfig.supervisor_notification
+    )?;
+    writeln!(
+        file,
+        "const HUBRIS_TASK_COUNT: usize = {};",
+        kconfig.tasks.len()
+    )?;
+
+    writeln!(
+        file,
+        "static HUBRIS_TASK_DESCS: [abi::TaskDesc; {}] = [",
```

Comment:
Nit – You could do
```rust
"static HUBRIS_TASK_DESCS: [abi::TaskDesc; HUBRIS_TASK_COUNT] = ["
```
since you defined `HUBRIS_TASK_COUNT` above

Reply:
Correct! Will fix.